### PR TITLE
FRONT-712: Extend ErrorContent with IllustrationPlaceholder's size & direction

### PR DESCRIPTION
**Pull request by DC-RomanKarpov**
Remove some unused styles

### CODE REVIEW

File: src/ui/components/ErrorContent/ErrorContent.scss [hunk 1]
```diff
@@ -32,13 +32,24 @@
...
+    }
+
     &_no-controls {
         display: block;
         padding: 20px 0;
     }
 
     &__title {
-        font-weight: 500;
+        font-weight: var(--g-text-header-font-weight);
```
Comment (ariser) [inline src/ui/components/ErrorContent/ErrorContent.scss:52]: we have this variable set to 500 in DC code
but isn't this is a component that DL also uses?
I don't know if they have the same override

**Review by ariser — COMMENTED** [review]
lgtm, but I suppose we should also show this to someone from DL team

File: src/ui/components/ErrorContent/ErrorContent.scss (hunk 1 above)
Comment (DC-RomanKarpov) [reply src/ui/components/ErrorContent/ErrorContent.scss:52]: Good point. Will remove. Thanks

**Review by mournfulCoroner — APPROVED** [review]

**Review by ariser — APPROVED** [review]

File: src/ui/components/ErrorContent/ErrorContent.js [hunk 2]
```diff
@@ -272,12 +276,19 @@ class ErrorContent extends React.PureComponent {
...
+                    className,
+                )}
+            >
                 <PlaceholderIllustration
                     name={imageName}
                     title={this.renderTitle()}
                     description={this.renderDescription()}
                     renderAction={() => this.renderActions()}
+                    size={size}
+                    direction={direction}
```
Comment (mournfulCoroner) [inline src/ui/components/ErrorContent/ErrorContent.js:291]: By default in `PlaceholderIllustration` direction switches to column in mobile view. This behavior may get lost if we pass direction in that way. I think you need to check here by DL.IS_MOBILE too

**Review by mournfulCoroner — CHANGES_REQUESTED** [review]

File: src/ui/components/ErrorContent/ErrorContent.js (hunk 2 above)
Comment (DC-RomanKarpov) [reply src/ui/components/ErrorContent/ErrorContent.js]: Thank you for the remark. Will add this behavior.

**Review by mournfulCoroner — APPROVED** [review]

**Review by ariser — CHANGES_REQUESTED** [review]

**Review by ariser — APPROVED** [review]

**Pull request merged**